pkg/webhook/webhook.go

```diff
@@ -0,0 +1,111 @@
+/*
+Copyright 2025 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package webhook
+
+import (
+	"context"
+	"crypto/tls"
+	"fmt"
+	"net/http"
+
+	"github.com/spf13/cobra"
+
+	"k8s.io/klog/v2"
+)
+
+var (
+	certFile                         string
+	keyFile                          string
+	kubeconfigFile                   string
+	port                             int
+	preventVolumeModeConversion      bool
+	enableVolumeGroupSnapshotWebhook bool
+)
+
+var CmdWebhook = &cobra.Command{
```

Comment:
should be most of this file be in cmd/webhook? The controller and the sidecar have their cmdline params defined in cmd/

Reply:
Agree. I reused the structure of the current validation webhook as far as possible, but having everything in `cmd/` is clearer in my opinion.